### PR TITLE
Link to official Project Loved website in lieu of a team listing

### PR DESCRIPTION
**Pull request by Walavouchey**
Making this pr for discussion. I'm not sure if this would be a correct change to make (even if temporary).

This would eliminate a lot of pointless work in keeping the listing on the wiki up to date with the project loved website. See discussion in https://github.com/ppy/osu-wiki/pull/7032 for rationale (this pr would also supersede it if merged).

We could maintain both at the same time with no additional effort, if a script checked the team listing on the loved website (https://loved.sh/team) and automatically sent a pr to the wiki, which I might do at some point.

This change can't really be made for the other team listings (i.e. only linking to group pages), since there's more metadata in those lists that the group pages don't support (area of focus, languages).

### CODE REVIEW

File: wiki/People/The_Team/Project_Loved_Team/en.md [hunk 1]
```diff
@@ -8,169 +8,16 @@ tags:
...
-- ![][flag_GB] [Pope Gadget](https://osu.ppy.sh/users/2288341)
-- ![][flag_SG] [Shoegazer](https://osu.ppy.sh/users/2520707)
-- ![][flag_US] [Toaph Daddy](https://osu.ppy.sh/users/7616811)
-
-### Video editors
-
-- ![][flag_GB] [Striiker](https://osu.ppy.sh/users/7291594)
-- ![][flag_US] [ThatsNotAName](https://osu.ppy.sh/users/9682904)
-- ![][flag_HU] [verto](https://osu.ppy.sh/users/2015300)
+The [Project Loved group page](https://osu.ppy.sh/groups/31) lists all of the team members. A more detailed member listing for all of the various roles within the Project Loved team, as well as previous members, can be found on the official [Project Loved website](https://loved.sh/team).
```
[the remark this answers is not in the document]
Comment (Walavouchey) [reply wiki/People/The_Team/Project_Loved_Team/en.md:13]: sure, it can be omitted, but care to elaborate?

**Review by peppy — APPROVED** [review]

**Comment by peppy** [comment]
I've been rolling my eyes each week when a new "update team" PR arrives and muting it. If there's anything else required from our end to make the team listing page usable as a replacement do let me know.

**Review by TicClick — APPROVED** [review]
on topic of the PR, I welcome it!

**Comment by TicClick** [comment]
> I'm not sure if this would be a correct change to make (even if temporary).

I think that linking to external sites is discouraged when they represent knowledge bases or anything similar. as there is already a lot of links to loved.sh for beatmap submission, I don't see any reason why the wiki shouldn't reference a more up to date list of users from there.

**Comment by cl8n** [comment]
> IIRC @cl8n has some work stashed.

for staff log, not group pages

**Pull request merged**